specutils/fitting/fitmodels.py

```diff
@@ -128,7 +128,9 @@ def find_lines_threshold(spectrum, noise_factor=1):
 
     # Threshold based on noise estimate and factor.
     uncertainty = spectrum.uncertainty
-    inds = np.where(np.abs(spectrum.flux) > (noise_factor*uncertainty.array) *
+    uncert_val = uncertainty.array if uncertainty is not None else 0
```

Comment:
```suggestion
    uncert_val = 0 if uncertainty is None else uncertainty.array 
```
(very minor style quibble)